src/screens/HomePage/HomePage.js

```diff
@@ -21,9 +19,8 @@ export default function HomePage() {
       <main>
         <div className="App">
           <Content>
-            {!isMobile && <CriteriaExplanation />}
+            <CriteriaExplanation />
```

Comment:
I assume this change is intentional, but it wasn't mentioned in the PR description so please confirm.

Reply:
Yep – we show above the map in all cases now. Before it was top on desktop only.